ynh-dev

```diff
@@ -379,9 +405,11 @@ EOF
                 fi
 
                 local DISTRO="$(lsb_release -s -c)"
+                
+                systemctl start yunohost-api
```

Comment:
That's weird. When working with the container, `yunohost-api` seems not started and it creates problems when developing the front.